### ISSUE TITLE
Continue skips while loop condition check

### ISSUE THREAD

**Issue by StylishTriangles** [algorand/pyteal#331]
### Subject of the issue

`Continue()` statement skips the check inside the while loop.

### Your environment

`pyteal 0.12.1`

### Steps to reproduce

Running the script below:
```python
from pyteal import *

i = ScratchVar(slotId=10)
ast = Seq(
    i.store(Int(0)),
    While(i.load() < Int(30)).Do(
        Seq(
            i.store(i.load() + Int(1)),
            Continue(),
        )
    ),
    Return(Int(1))
)

comp = compileTeal(ast, Mode.Application, version=6)

print(comp)
```

...prints the following TEAL:
```
#pragma version 6
int 0
store 10
load 10
int 30
<
bz main_l2
main_l1:
load 10
int 1
+
store 10
b main_l1
main_l2:
int 1
return
```

Note, that `main_l1:` is just after the condition check, therefore the loop never terminates (it technically does after 18446744073709551615 iterations, but that's an edge case I'm not concerned about).

### Expected behaviour

I expect the PyTEAL compiled code to actually terminate, to better portray what I mean, I have created the following script in Python:

```python
i = 0
while i < 30:
    i += 1
    continue

# control reaches here, therefore the program halts 🥳 
```

### Actual behaviour

∞ lööp

### Extra info
`Continue()` works as expected inside the `For` loop

**Comment by ahangsu** [comment]
Fixed in #332

**Comment by StylishTriangles** [comment]
Thanks a lot for the swift fix!
